# Action to be performed
string action

# List of condition predicates
ownage_bot/PredicateMsg[] conditions

# Deontic operator type
string detype

# Truth value between 0.0 and 1.0
float32 truth